Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -73,7 +73,20 @@ public ProtocolVersion ProtocolVersion
 			set;
 		}
 
-		public Network Network { get; private set; }
+		private Network _network = null;
+		public Network Network
+		{
+			get { return GetNetwork(); }
+			private set { this._network = value; }
+		}
+
+		private Network GetNetwork()
```

Comment:
Yes, I started that way and thought it was ugly that way too.
Hold off on this, I'll see if I can clean up.